### DISCUSSION TITLE
Switch off automatic node movement

### DISCUSSION THREAD

**Discussion by GenerallyDisjointed** [freeplane/freeplane#711] [Help] [answered]
Hi everyone, I don't know quite what to search on to see if this has already been asked.

When I'm moving nodes around, my other nodes are moving around in an erratic way, is there a way I can switch off the auto movement of them?

**Comment by abc16361** [comment, marked as answer]
Just set the node as 'Free positioned node' (right click context menu), then it won't affect others.

**Reply by GenerallyDisjointed** [reply]
Thankyou - perfect

**Comment by dpolivaev** [comment]
To stop what you call "erratic movement"

Navigate->Goto root
Navigate->Fold->Unfold all
Edit->Reset node position

Let Freeplane arrange your node positions. If you want to arrange all node positions yourself, Freeplane is probably not your tool of choice.

**Reply by GenerallyDisjointed** [reply]
Thankyou! I'm using Freeplane for my mba, a lot of the time when I'm moving nodes, adjacent ones will either wrap around to the opposite side, or fly off somewhere. Been using Freeplane for a very long time, it's fantastic - best mindmapping software there is
